UVAtlas/inc/UVAtlas.h

```diff
@@ -127,7 +131,7 @@ namespace DirectX
     //                 maximum number of charts was too low, this gives the minimum
     //                 number of charts needed to create an atlas.
 
-    HRESULT __cdecl UVAtlasCreate(
+    HRESULT _CDECL UVAtlasCreate(
```

Comment:
i guess I'd prefer if we didn't preface our custom types with _ given that it is normally reserved for system types.

Reply:
Can change. `JNT_CDECL`?

Reply:
that seems better to me